cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -1406,22 +1237,23 @@ def configure_apiserver(etcd_connection_string):
     audit_policy_path = audit_root + '/audit-policy.yaml'
     audit_policy = hookenv.config('audit-policy')
     if audit_policy:
-        write_audit_config_file(audit_policy_path, audit_policy)
+        write_config_file(audit_policy_path, audit_policy)
```

Comment:
`write_config_file` seems a bit too generic. If you really want to avoid having it tied to the audit configs, then I'd go with something like `write_file_with_autogenerated_header`.